oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/service/ServiceTraffic.java

```diff
@@ -92,7 +92,7 @@ public class ServiceTraffic extends Metrics {
     @Setter
     @Getter
     @Column(columnName = LAYER)
-    @BanyanDB.SeriesID(index = 1)
+    @BanyanDB.SeriesID(index = 0)
```

Comment:
What is the reason of the layer first?

Reply:
Searching SeriesID supports prefix scanning. OAP uses finding by `layer` a lot. Based on that, moving the `layer` to the first place will improve the search performance a bit.

Reply:
Make sense. Let's add a little comments to seriesID index.
We should indicate the index matters for prefix searching.

Reply:
Done